configuration/configmgr/configmgrlib/mod_template_support/Variables.cpp

```diff
@@ -90,16 +90,20 @@ std::string Variables::doValueSubstitution(const std::string &value) const
     std::size_t bracesStartPos = result.find("{{");
     bool done = bracesStartPos == std::string::npos;
 
+    // {{hwips[{{numsystems}}-{{ccnum}}]}}
+    // ^                                ^
+    // 0                                33
```

Comment:
was this comment supposed to be left in?

Reply:
Was a debugging aid when stepping through the code. Removed.